x-pack/plugins/maps/common/elasticsearch_util/elasticsearch_geo_utils.test.js

```diff
@@ -397,12 +397,10 @@ describe('createExtentFilter', () => {
       minLon: -89,
     };
     const filter = createExtentFilter(mapExtent, geoFieldName);
```

Comment:
You could just do a destructuring assignment to make it more succinct:
```
const { geo_bounding_box } = createExtentFilter(mapExtent, geoFieldName);
```

Or if you really wanted, just grab `location`:

```
const { geo_bounding_box: { location }}...
```